anesthetic/plot.py

```diff
@@ -488,8 +488,7 @@ def contour_plot_2d_sample_kde(ax, data_x, data_y, *args, **kwargs):
                        ymin=ymin, ymax=ymax)
     pdf /= pdf.max()
     p = sorted(pdf.flatten())
-    m = numpy.cumsum(p)
-    m /= m[-1]
+    m = numpy.arange(1, len(p)+1)/(len(p)+1)
```

Comment:
Why does this need to be different from `grid_kde`?

Reply:
This was the subtle bug that was causing the difference in 2D plot widths between kde and fastkde. Samples from the distribution (as in `contour_plot_2d_sample_kde`) distribute uniformly across the probability range. Gridded point (as in `contour_plot_2d_grid_kde`) need to be `cumsum`ed across the range.

Reply:
Ahh, thanks, that makes perfect sense.